.gitignore

```diff
@@ -9,4 +9,24 @@ crashes/*
 suppressions
 main
 build
-FuzzyVM
\ No newline at end of file
+FuzzyVM
+config.toml
+
+# Git Ignore file
+
+# mac
+# General
+.DS_Store
```

Comment:
I would prefer these not to be in the `.gitignore` file as I'm not on OSX, but if they are bothering you too much, and you don't want to exclude them globally, it's fine to leave them in